students/shawn/class_03/strformat_lab.py

```diff
@@ -126,7 +126,6 @@ def task_six(fnam):
 
     # create formats for each column
     colspace=3
-    #
     fmts=[]
     for i in range(len(head)):
         sym= '<' if i < len(head) - 1 else '>'
```

Comment:
If you're stuck on this one, check out this walk through on string formatting: 
https://www.blog.pythonlibrary.org/2017/02/08/new-in-python-formatted-string-literals/